addon/server.js

```diff
@@ -244,7 +244,7 @@ export default class Server {
    * @public
    */
   shouldTrackRequests() {
-    return typeof this.trackRequests !== 'undefined' ? this.trackRequests : false;
+    return this.trackRequests ? this.trackRequests : false;
```

Comment:
Isn't this essentially the same as either `return !!this.trackRequests` or `return Boolean(this.trackRequests)`?  I think either of those are a bit more idiomatic in this case...

Reply:
Agreed, either of those are better. I'll update.